go/vt/schemadiff/table.go

```diff
@@ -1823,6 +1823,18 @@ func (c *CreateTableEntity) validate() error {
 			}
 		}
 	}
+	// validate all columns referenced by foreign key constraints do in fact exist
+	for _, cs := range c.CreateTable.TableSpec.Constraints {
```

Comment:
On the other hand it will increase code nesting and conditionals. I feel like these are two very different kinds of constraints; for the handful of items that will be in that slice I prefer the readability.

Reply:
Sounds good, I doubt this ever would be a performance bottleneck :smile:.